forms-flow-components/src/components/SvgIcons/index.tsx

```diff
@@ -240,6 +240,27 @@ export const CloseIcon = ({
   </svg>
 );
 
+export const DeleteIcon = ({ color = baseColor, onClick, ...props }) => (
+  <svg xmlns="http://www.w3.org/2000/svg"
+    width="14"
+    height="16"
+    viewBox="0 0 14 16"
+    fill="none"
+    onClick={onClick}
+    {...props}
+  >
+      <path d="M7 8L7 11" 
+    stroke="#253DF4" 
+    strokeWidth="2" 
+    strokeLinecap="round" />
+    <path d="M2.41176 3.94737L3.82353 15H10.1765L11.5882 3.94737M1 3.94737H4.52941M13 3.94737H9.47059M4.52941 3.94737L5.23529 1H8.76471L9.47059 3.94737M4.52941 3.94737H9.47059" 
+    stroke="#253DF4" 
```

Comment:
pls check whether we can use 'color' here instead of #253df4  ? in 2 places

Reply:
Updated